subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AndroidSantaTrackerCachingSmokeTest.groovy

```diff
@@ -684,8 +679,8 @@ class AndroidPluginExpectations {
         ':wearable:generateDebugResValues': FROM_CACHE,
         ':wearable:generateDebugResources': UP_TO_DATE,
         ':wearable:javaPreCompileDebug': FROM_CACHE,
-        ':wearable:kaptDebugKotlin': SKIPPED,
-        ':wearable:kaptGenerateStubsDebugKotlin': SKIPPED,
+        ':wearable:kaptDebugKotlin': FROM_CACHE,
+        ':wearable:kaptGenerateStubsDebugKotlin': FROM_CACHE,
```

Comment:
Do we know why this changed? There are [no Kotlin files in `:wearable`](https://github.com/gradle/santa-tracker-android/tree/a6dc397049c41d58bc5677b15aad934c57636b36/wearable/src/main), so these should be `SKIPPED` still, no?

Reply:
I don't know what is the reason for these tasks to be from the cache now and not skipped. Do you have some insights?

And there are Kotlin files in `wearable`: https://github.com/gradle/santa-tracker-android/blob/a6dc397049c41d58bc5677b15aad934c57636b36/wearable/src/main/java/com/google/android/apps/santatracker/ChristmasCountdownProviderService.kt

This should be some change on the santa tracker project, though I don't know which one.

Reply:
What a vaguely remember is that folks use `src/main/java` as the only source directory. I don't even know if the Kotlin plugin adds this or if there is some configuration on the santa tracker build.

Reply:
There is `:wearable:compileDebugKotlin': FROM_CACHE` as well, so it seems like those things are compiled.

Reply:
It may be because we removed these properties: https://github.com/gradle/santa-tracker-android/pull/17/files#diff-3d103fc7c312a3e136f88e81cef592424b8af2464c468116545c4d22d6edcf19L46-L48